meta-balena-common/classes/image-balena.bbclass

```diff
@@ -365,6 +361,7 @@ python os_release_extra_data() {
         f.writelines(extra_data)
 }
 
+ROOTFS_POSTPROCESS_COMMAND_remove = "zap_empty_root_password"
```

Comment:
I'm afraid that actually removes all semi-colons present in the string. The initial version does keep an extra semi-colon but it has no functional effect.

Reply:
Hmm, I only suggested the change because I saw the build fail to parse `ROOTFS_POSTPROCESS_COMMAND` on one of the older device types, likely using an older yocto version. I guess we will see if it pops up again.

Reply:
you're right, this will only work on Dunfell upwards. Unfortunately the only way for older versions to work is to backport a Poky patch (https://github.com/balena-os/poky/pull/1). Given that warrior is EOL it's a good moment to force those DTs still on older Yocto versions to update, and if they really can't they will need to use a forked Poky.